src/confcom/azext_confcom/tests/latest/test_confcom_scenario.py

```diff
@@ -48,6 +49,7 @@ class MountEnforcement(unittest.TestCase):
                 ]
             },
             {
+                "name": "mcr.microsoft.com/cbl-mariner/distroless/python:3.9-nonroot",
```

Comment:
I would switch all of these to use azure linux since it's the latest version of mariner (with the name change for added confusion)